src/placement/jump_map.c

```diff
@@ -341,9 +341,18 @@ obj_remap_shards(struct pl_jump_map *jmap, uint32_t layout_ver, struct daos_obj_
 			get_target(root, layout_ver, &spare_tgt, crc(key, rebuild_key),
 				   dom_used, dom_full, dgu->dgu_used, tgts_used, shard_id,
 				   allow_status, fdom_lvl, &spares_left, &spare_avail);
-			D_ASSERT(spare_tgt != NULL);
-			D_DEBUG(DB_PL, "Trying new target: "DF_TARGET"\n",
-				DP_TARGET(spare_tgt));
+			if (layout_ver > 1) {
```

Comment:
Your commit message says "only check ... layout version > 0", but here the check is "layout_ver > 1"?

Reply:
ah, this should > 0. Thanks!